bundle/render_templates

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
+set -x
+
+env
+
+curl -L -o ./jq https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 && chmod 0755 ./jq
+curl -L -o ./yq https://github.com/mikefarah/yq/releases/download/v4.16.1/yq_linux_amd64 && chmod 0755 ./yq
+
+
+# get the required environment variables that are set when merging change sets
+source render_vars
+
+# Check for environment variables pertaining to images
+if [[ -z "$CI_UPSTREAM_VERSION_SANITIZED" ]] ||
+   [[ -z "$CI_TOPOLOGY_AWARE_LIFECYCLE_MANAGER_UPSTREAM_URL" ]] ||
+   [[ -z "$CI_TOPOLOGY_AWARE_LIFECYCLE_MANAGER_UPSTREAM_COMMIT" ]] ||
+   [[ -z "$CI_TOPOLOGY_AWARE_LIFECYCLE_MANAGER_BUILD_INFO_JSON" ]]; then
+  echo "ERROR: All required environment variables not loaded"
+  echo "    CI_UPSTREAM_VERSION_SANITIZED"
+  echo "    CI_TOPOLOGY_AWARE_LIFECYCLE_MANAGER_UPSTREAM_URL"
+  echo "    CI_TOPOLOGY_AWARE_LIFECYCLE_MANAGER_UPSTREAM_COMMIT"
+  echo "    CI_TOPOLOGY_AWARE_LIFECYCLE_MANAGER_BUILD_INFO_JSON"
+  exit 3
```

Comment:
In the error cases, the downloaded jq and yq commands are being left in place, because they're only cleaned up at the end of the script in the success path. I'd suggest using a tmpdir created by mktemp for the downloaded files (and any other temporary work products in the future) and adding a cleanup function that deletes it, with a "trap cleanup EXIT" to ensure it always gets cleaned up